src/pages/contributors/alexander-brandes.adoc

```diff
@@ -0,0 +1,81 @@
+
+When he is not pursuing open-source endeavors, Alexander is a passionate musician and has been playing the trumpet for more than 15 years, primarily playing jazz and film music. In addition to being the lead trumpet in a band, he also teaches beginners how to play an instrument.
+
+== What is your background prior to contributing to Jenkins?
+
+When I was 13, my friends and I discovered the sandbox game _Minecraft_.
+Eventually, we created our own Minecraft server without any clue about what we were doing.
+The game is written in Java, and can be modified or extended with plugins.
+Even though we weren't fully aware of what we were doing, we had a great time with writing our first plugin.
+It wasn't any good, considering what I know now, but the few people who played on the server liked it a lot, which still means a lot to me almost a decade later.
+In 2020, we had more than 5k concurrent players every day and grew a Discord community made up of around ~250,000 active members, becoming one of the largest Minecraft servers worldwide.
+
+Ultimately, this hobby project shaped a major part of me over the past decade.
+Throughout the years, I continued learning more about Java, containers, and networking as the single server evolved into a continuously growing network.
+Everything I needed to learn, I had to teach myself since I had no technical background at all.
+The knowledge acquired helped me tremendously when getting into open source and development. 
+I still enjoy looking back at what started as a small project and turned into something thousands of people have a great time on every day.
+It is absolutely inspiring for me to see.
+
+Before I started contributing to Jenkins, I had already contributed to smaller open-source projects.
+My contributions primarily focused on Java-based projects and documentation improvements.
+These contributions, paired with continuous learning of new languages and technologies, helped improve my knowledge and advance future contributions.
+
+== How long have you been using Jenkins?
+
+I have been using Jenkins since 2019.
+
+== Why choose Jenkins over other projects?
+
+Jenkins maintains a notably low barrier of entry, which makes it attractive to contribute to, regardless of the massive plugin ecosystem around Jenkins.
+What I liked the most when I started contributing to Jenkins, is that you can start with small pull requests for plugins, and move on to bigger contributions when you feel more comfortable with the codebase and ecosystem.
+
+Moreover, Jenkins is something I had used before I started contributing.
+Therefore, I already had a certain level of experience as an end user, before becoming a contributor.
+That's a key point which made it easier for me to contribute, because I've already been a user of the project, instead of a contributing to a project totally unfamiliar to me.
```

Comment:
Should "a contributing" be "a contributor" or remove 'a' (before 'contributing')

Reply:
Thanks for catching that @alyssat! I thought I had cleaned that up but may have missed it when I was converting things over to adoc.